### PR TITLE
View for confirmed events list

### PR DESCRIPTION
**Pull request by lkudyba**
@quirk0o @mhib @jtkpiotr @adamzima

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -87,4 +91,8 @@ def span_param
   def time_interval_rfc3339
     span_param.to_rfc3339
   end
+
+  def since_beginning_of_epoch
```
Comment (jtkpiotr) [inline app/controllers/events_controller.rb]: I guess it should be a TimeInterval's method

File: app/controllers/events_controller.rb [hunk 2]
```diff
@@ -61,6 +61,10 @@ def confirm
     Event.confirm_or_create(conference_room_id, event_id)
   end
 
+  def confirmed
+    @confirmed_events = google_event_client.confirmed_events TimeInterval.since_beginning_of_epoch.to_rfc3339
```
Comment (adamzima) [inline app/controllers/events_controller.rb]: ( ) - you can copy them from here

File: config/routes.rb [hunk 3]
```diff
@@ -8,6 +8,7 @@
   post 'notify/:conference_room_id' => 'notification#receive', as: :notifications
   get 'conference_rooms/:conference_room_id/events' => 'events#room_index'
   post 'conference_rooms/:conference_room_id/events/:event_id/confirm' => 'events#confirm', as: :confirmation
+  get 'events/confirmed' => 'events#confirmed'
```
Comment (adamzima) [inline config/routes.rb]: Can we put it to resources section?

File: app/views/events/confirmed.html.erb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<h1>Confirmed events:</h1>
```
Comment (adamzima) [inline app/views/events/confirmed.html.erb:1]: Why do we need this view? Should we have special access level for it?

File: app/controllers/events_controller.rb (hunk 2 above)
Comment (quirk0o) [reply app/controllers/events_controller.rb]: ⛄

File: app/views/events/confirmed.html.erb (hunk 4 above)
Comment (quirk0o) [reply app/views/events/confirmed.html.erb:1]: Przemek the Project Owner desired it.

File: app/controllers/events_controller.rb (hunk 2 above)
Comment (jtkpiotr) [inline app/controllers/events_controller.rb]: thanks @adamzima, I couldn't find mine :/

Comment (lkudyba) [reply app/controllers/events_controller.rb]: thanks @adamzima, I couldn't find mine :/

**Review by mhib — APPROVED** [review]

**Review by madejejej — APPROVED** [review]

**Pull request merged**